src/ceph-disk/ceph_disk/main.py

```diff
@@ -4524,9 +4531,35 @@ def list_devices():
     return devices
 
 
+def list_zfs():
+    try:
+        out, err, ret = command(
+            [   'zfs',
+                'list',
+                '-o', 'name,mountpoint'
+            ]
+           )
+    except subprocess.CalledProcessError as e:
+        LOG.info('zfs list -o name,mountpoint '
+           'fails.\n (Error: %s)' % e)
+        return 1
+    lines = out.splitlines()
+    for line in lines[2:]:
+        vdevline = line.split()
+        if os.path.exists(vdevline[1]+'/active'):
+            elems = vdevline[1].split('/')
+            print(vdevline[0]+ " ceph data, active, cluster ceph, "+elems[5]+
```

Comment:
and there are other places that violates pep8 in this change, could you run this file with `pep8`, and make sure it's pep8 warning/errors free?